package.json

```diff
@@ -785,7 +787,7 @@
     "react-router-redux": "^4.0.8",
     "react-shortcuts": "^2.0.0",
     "react-sizeme": "^2.3.6",
-    "react-syntax-highlighter": "^5.7.0",
+    "react-syntax-highlighter": "^15.3.1",
```

Comment:
The existing dependency paths are:

```
react-syntax-highlighter@5.8.0 > highlight.js@9.12.0
react-syntax-highlighter@5.8.0 > lowlight@1.9.1 > highlight.js@9.12.0
```

I needed to upgrade the (years out-of-date) `react-syntax-highlighter` package so I could bump its transitive dependencies of `highlight.js` to `^10.4.1`.

@elastic/apm-ui, you are the only consumer of this package in Kibana. I read the [`react-syntax-highlighter` changelog](https://github.com/react-syntax-highlighter/react-syntax-highlighter/blob/master/CHANGELOG.MD) and updated `DatabaseContext` & `Context` components accordingly. The unit tests are passing but I couldn't figure out how to access these in a manual smoke test. If you could please take a look it would be much appreciated!